addons/website_partner/models/res_partner.py

```diff
@@ -10,9 +10,18 @@ class ResPartner(models.Model):
 
     website_description = fields.Html('Website Partner Full Description', strip_style=True, sanitize_overridable=True, translate=html_translate)
     website_short_description = fields.Text('Website Partner Short Description', translate=True)
+    website_published = fields.Boolean(tracking=True)
```

Comment:
I don't think you should track this as it's context-dependent. It doesn't really make sense to track "is this record published 'on the current website'" when we don't know what the current website was at the time.

I would just track `is_published`, and potentially `website_id` where relevant.

----------------------

On the technical side one issue for example is that if you write with some forced website in context for any reason, we'll reach the end of the transaction and see a different value for `website_published` because it was computed with two different contexts.